scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OTargetEncoderMOJOModel.scala

```diff
@@ -42,7 +42,7 @@ class H2OTargetEncoderMOJOModel(override val uid: String)
 
   @transient private lazy val orderOfInputColumns = {
     val mojoModel = Utils.getMojoModel(getMojo()).asInstanceOf[TargetEncoderMojoModel]
-    val indexes = mojoModel._teColumnNameToIdx
+    val indexes = mojoModel._columnNameToIdx
```

Comment:
interesting, I didn't think that users would access those Mojo properties directly.
What do you need this one for exactly? To know the order of the predictions columns?

Reply:
For the order of input columns. SW user provides the list of `inputColumns` in a arbitrary order. And in the same order, the result should be return. I remember TE MOJO was not returning predictions in an expected order (training order). `_teColumnNameToIdx` was a solution for ordering in [the SW case](https://github.com/h2oai/sparkling-water/pull/2314/files/2350260b812f11de283547bbd219cfff8b0c9b87#diff-cec96bfbb0969ede0661e2aaee3d9915R54).

Reply:
Ok, FYI the predictions from the Mojo should always follow the column order by index.
Maybe in a previous version, this was not the case, but it's been the case even before my refactoring.